app/models/concerns/new_notification.rb

```diff
@@ -0,0 +1,16 @@
+module NewNotification
+  extend ActiveSupport::Concern
+  included do
+    after_create :notify
+  end
+
+  def notify
+    NotificationSystemWorker.perform_async(model_name.to_s, id)
+  end
+
+  def object_for_notification
```

Comment:
true, would look into that, we shouldn't be slowing down the db, because of some new content and stuff